frontend/src/pages/edItVault/vaultRename/VaultRenamePage.tsx

```diff
@@ -0,0 +1,87 @@
+import { useTranslation } from 'react-i18next';
+import { PageHeader } from '../../../ui/page/PageHeader';
+import { PageHeaderBackButton } from '../../../ui/page/PageHeaderBackButton';
+import { PageHeaderTitle } from '../../../ui/page/PageHeaderTitle';
+import { PageSlice } from '../../../ui/page/PageSlice';
+import { VStack } from '../../../lib/ui/layout/Stack';
+import { FieldValues, useForm } from 'react-hook-form';
+import { zodResolver } from '@hookform/resolvers/zod';
+import { Text } from '../../../lib/ui/text';
+import {
+  ButtonWithBottomSpace,
+  InputField,
+  InputFieldWrapper,
+} from './VaultRenamePage.styles';
+import { z } from 'zod';
+
+const renameSchema = z.object({
+  vaultName: z.string().min(1, 'vault_rename_page_name_error').max(50),
+});
+
+const VaultRenamePage = () => {
+  const {
+    register,
+    handleSubmit,
+    formState: { errors, isValid, isDirty },
+  } = useForm({
+    resolver: zodResolver(renameSchema),
+    mode: 'onBlur',
+  });
+
+  const onSubmit = (data: FieldValues) => {
+    // TODO: @antonio to implement logic after UI is done
+    console.log('## data', data);
```

Comment:
looks like this PR is not complete

Reply:
to implement service login in a separate PR